sdks/uniswapx-sdk/src/builder/V3DutchOrderBuilder.test.ts

```diff
@@ -364,13 +406,51 @@ describe("V3DutchOrderBuilder", () => {
           ).toThrow("Invariant failed: nonce not set");
     });
 
+    it("Throw if startingBaseFee not set", () => {
```

Comment:
Maybe add a test for when `startingBaseFee` and  `adjustmentPerGweiBaseFee` are non-zero (e.g. `1e9`)